pkg/url/url.go

```diff
@@ -248,17 +249,20 @@ func ListPushed(client *occlient.Client, componentName string, applicationName s
 func ListPushedIngress(client *kclient.Client, componentName string) (URLList, error) {
 	labelSelector := fmt.Sprintf("%v=%v", componentlabels.ComponentLabel, componentName)
 	klog.V(4).Infof("Listing ingresses with label selector: %v", labelSelector)
-	ingresses, err := client.ListIngresses(labelSelector)
+
+	ingressList, err := client.ListIngresses(labelSelector)
 	if err != nil {
-		return URLList{}, errors.Wrap(err, "unable to list ingress names")
+		return URLList{}, fmt.Errorf("failed to list ingresses %w", err)
 	}
 
 	var urls []URL
-	for _, i := range ingresses {
-		a := getMachineReadableFormatIngress(i)
-		urls = append(urls, a)
+	for _, i := range ingressList {
+		if i.NetworkingV1Ingress != nil {
+			urls = append(urls, getMachineReadableFormatIngressNetworkingV1(*i.NetworkingV1Ingress))
```

Comment:
Again I feel that we should keep `getMachineReadableFormatIngress` as-is and have the logic for checking `NetworkingV1Ingress` in that function instead of doing it in `ListPushedIngress`.

Reply:
I will make this change right away